intro-Linux/ch_getting_started.tex

```diff
@@ -128,10 +144,10 @@ \subsubsection{Changing your prompt using ``\$PS1''}
 ~/test $
 \end{prompt}
 
-Note that ``\tilde'' is short representation of your home directory.
+Note that \verb|~| is short representation of your home directory.
```

Comment:
Does it still render correctly when you use `~` rather than `\tilde`?

Reply:
I'm using `\verb|~|`, because if just `~` is used, it sticks to the next word.

Reply:
OK, but why prefer `~` over `\tilde`? Just wondering... "less typing" is a good reason to me ;)